code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -44,3 +44,263 @@
 /mob/living/simple_animal/hostile/retaliate/clown/attack_hand(mob/living/carbon/human/M)
 	..()
 	playsound(src.loc, 'sound/items/bikehorn.ogg', 50, 1)
+
+/mob/living/simple_animal/hostile/retaliate/clown/lube
+	name = "Living Lube"
+	desc = "A puddle of lube brought to life by the honkmother."
+	icon = 'icons/mob/clown_mobs.dmi'
+	icon_state = "lube"
+	icon_living = "lube"
+	turns_per_move = 1
+	response_help = "dips a finger into"
+	response_disarm = "gently scoops and pours aside"
+	emote_see = list("bubbles", "oozes")
+	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/particle_effect/foam)
+
+/mob/living/simple_animal/hostile/retaliate/clown/lube/Life()
+	. = ..()
+	var/turf/open/OT = get_turf(src)
+	if(isopenturf(OT))
+		OT.MakeSlippery(TURF_WET_LUBE, 100)
+
+/mob/living/simple_animal/hostile/retaliate/clown/banana
+	name = "Clownana"
+	desc = "A fusion of clown and banana DNA birthed from a botany experiment gone wrong."
+	icon = 'icons/mob/clown_mobs.dmi'
+	icon_state = "banana tree"
+	icon_living = "banana tree"
+	response_help = "pokes"
+	response_disarm = "peels"
+	response_harm = "peels"
+	turns_per_move = 1
+	speak = list("HONK", "Honk!", "YA-HONK!!!")
+	emote_see = list("honks", "bites into the banana", "plucks a banana off its head", "photosynthesizes")
+	maxHealth = 120
+	health = 100
+	speed = -10
+	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/gibspawner/human, /obj/item/soap, /obj/item/seeds/banana)
+	var/banana_time = 20
+
+/mob/living/simple_animal/hostile/retaliate/clown/banana/Life()
+	. = ..()
+	if(banana_time < world.time)
+		var/turf/T = get_turf(src)
+		var/list/adjacent =  T.GetAtmosAdjacentTurfs(1)
+		new /obj/item/grown/bananapeel/(pick(adjacent))
+		banana_time = world.time + rand(30,60)
+
+/mob/living/simple_animal/hostile/retaliate/clown/honkling
+	var/datum/reagent/funjuice = "laughter"
+	name = "Honkling"
+	desc = "A divine being sent by the Honkmother to spread joy. It's not dangerous, but it's a bit of a nuisance."
+	icon = 'icons/mob/clown_mobs.dmi'
+	icon_state = "honkling"
+	icon_living = "honkling"
+	turns_per_move = 1
+	speed = -10
+	harm_intent_damage = 1
+	melee_damage_lower = 1

```

Comment:
could you reduce both this and melee_damage_upper to 0, or does the code/game break if that happens?

Reply:
it uses friendly verbs if you're 0

Reply:
I also have a proc set to check when it attacks, and it doesn't activate when it's set to 0